Kconfig

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2017 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "MCUBOOT configuration"
+
+config ZEPHYR_BASE
+	string
+	option env="ZEPHYR_BASE"
+
+config APPLICATION_BASE
+	string
+	option env="PROJECT_BASE"
+
+source "$ZEPHYR_BASE/Kconfig.zephyr"
+
+# this is shadow entry for selections making
+config MCUBOOT_FORCE_CONFIG_SELECTION
+	bool
+	default y
+	# avoid MCUB-90 bug on nRF5x targets
+	select FLASH_PAGE_LAYOUT if SOC_FLASH_NRF5
```

Comment:
Why does this need to be selected here instead of being done in Zephyr's own Kconfig files?

Reply:
Actually would safe a lot of hassle (currently) :+1:

Reply:
This came up a while back; I believe @nvlsianpu may have been concerned about the additional size this added to the flash library:

https://github.com/zephyrproject-rtos/zephyr/pull/1594#pullrequestreview-64524847

Reply:
@carlescufi I was intended to avoid forcing to use this extension in general (for all zephyr nRF5 targets).
I wouldn't add mcuboot dependency into zephyr (that is what you mean?).

Reply:
After discussion with Carles I will remove this, and I will make FLASH_PAGE_LAYOUT selected by default for nRF5 targets in zephyr.